db/compaction_picker.h

```diff
@@ -84,15 +84,15 @@ class CompactionPicker {
   void ReleaseCompactionFiles(Compaction* c, Status status);
 
   // Returns true if any one of the specified files are being compacted
-  bool FilesInCompaction(const std::vector<FileMetaData*>& files);
+  bool IfFilesInCompaction(const std::vector<FileMetaData*>& files);
```

Comment:
Just saying: This name feels weird when used in if/while conditions. For example: `if (FilesInCompaction(...))` is preferable to `if (IfFilesInCompaction(...))`. But the name looks good when used in a return statement like `return IfFilesInCompaction(...)`. I understand there is a no one-good-way to name them. 
That said, we are not using a consistent way of naming boolean functions. For example: PickFileToCompact, ExpandInputsToCleanCut are also returning bools, without a question (is, are, ...) at the beginning. 

I don't have a preference one way or the other, but might be good to follow one convention throughout the code base.